public/pages/MonitorDetails/containers/utils/helpers.js

```diff
@@ -12,9 +12,10 @@
  *   express or implied. See the License for the specific language governing
  *   permissions and limitations under the License.
  */
-
+export const NAME_REGEX = RegExp('^[a-zA-Z0-9._-]+$');
+export const MAX_INTERVAl_LENGTH_MINUTES = 10080;
```

Comment:
Wondering, so this will allow a weeks worth of data per interval? Was this agreed upon with others or just determined by you?

Reply:
This was agreed upon with other team-members but with the understanding that it could change as more user data is collected and as the AD model changes